### PR TITLE
fix: Provide fee asset

### PR DESCRIPTION
**Pull request by AndonMitev**

### CODE REVIEW

**Review by skeremidchiev — APPROVED** [review]

**Comment by monokh** [comment]
Going to need a description on this one. 
Why is this a requirement now?

**Comment by AndonMitev** [comment]
> Going to need a description on this one. Why is this a requirement now?

Yes so, Astroport dapp works only with UST for fee asset and we were passing Luna soo all dapp interactions were failing, so now wallet is providing the fee asset which is passed from dapp to wallet-core and wallet-core provide correct fee asset to chainify

File: src/store/actions/sendTransaction.ts [hunk 1]
```diff
@@ -30,6 +31,7 @@ export const sendTransaction = async (
     amount: BigNumber;
     data: string;
     fee: number;
+    feeAsset: string;
```
Comment (monokh) [inline src/store/actions/sendTransaction.ts:34]: Can it be optional so we don't need to specify it when its not required?

**Review by monokh — APPROVED** [review]

File: src/store/actions/sendTransaction.ts (hunk 1 above)
Comment (AndonMitev) [reply src/store/actions/sendTransaction.ts:34]: yes

**Pull request merged**